validator/sawtooth_validator/database/indexed_database.py

```diff
@@ -109,6 +109,16 @@ def __len__(self):
         with self._lmdb.begin(db=self._main_db) as txn:
             return txn.stat()['entries']
 
+    def len(self, index=None):
```

Comment:
I don't think you should shadow the built-in function here.  I would suggest naming this either `size` or `length` and then calling it with the default args in the `__len__` implementation.